include/api.php

```diff
@@ -1116,12 +1111,7 @@ function api_statuses_update($type)
 		throw new ForbiddenException();
 	}
 
-	$user_info = api_get_user($a);
```

Comment:
I have the the feeling as if ```api_get_user``` is also used to check if the user was authenticated. See the code here: https://github.com/friendica/friendica/blob/develop/include/api.php#L588 

AFAIK this code will be executed whenever the function is called like above and the user hadn't authenticated yet.

Reply:
Nice catch, I should just have removed the variable assignment.